### PR TITLE
DSM: Add CheckRoundIcon  + Add Green10 + Add SuccessHelper

### PR DESCRIPTION
**Pull request by tomglvng**
Capture du 2021-01-21 14-24-32

---

In the Event Subscription page, the helper is now correctly displayed under the input:

Screenshot_2021-01-22_09-34-18

### CODE REVIEW

**Review by rouliane — APPROVED** [review]

File: akeneo-design-system/src/icons/SuccessIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {IconProps} from './IconProps';
+
+const SuccessIcon = ({title, size = 24, color = 'currentColor', ...props}: IconProps) => (
```
Comment (ValentinMumble) [inline akeneo-design-system/src/icons/SuccessIcon.tsx:4]: What do you think about naming it `CheckRoundIcon` to follow the same pattern as `InfoIcon` & `InfoRoundIcon`?

File: akeneo-design-system/src/icons/index.ts [hunk 2]
```diff
@@ -108,3 +108,4 @@ export * from './UploadIcon';
 export * from './ValueIcon';
 export * from './ViewIcon';
 export * from './WaveIcon';
+export * from './SuccessIcon';
```
Comment (ValentinMumble) [inline akeneo-design-system/src/icons/index.ts:111]: To put it at the correct order here https://akeneo.github.io/akeneo-design-system/13764/?path=/docs/guidelines-iconography--standard, you have to order it alphabetically 😉

**Review by ValentinMumble — APPROVED** [review]

**Review by olivierpontier — APPROVED** [review]

File: akeneo-design-system/src/icons/SuccessIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  <svg viewBox="0 0 24 24" width={size} height={size} {...props}>
+    {title && <title>{title}</title>}
+    <g
+      id="Icons-/-success"
```
Comment (tseho) [inline akeneo-design-system/src/icons/SuccessIcon.tsx:8]: I think we can remove the `id` from the svg elements, it's not needed.

File: akeneo-design-system/src/icons/SuccessIcon.tsx (hunk 1 above)
Comment (tseho) [reply akeneo-design-system/src/icons/SuccessIcon.tsx]: btw, https://www.notion.so/akeneo/Iconography-40ec9ee224094bd0906e05549b901ff1 is not up-to-date

**Review by tseho — APPROVED** [review]

File: akeneo-design-system/src/icons/SuccessIcon.tsx (hunk 1 above)
Comment (tseho) [reply akeneo-design-system/src/icons/SuccessIcon.tsx]: @tomglvng You were too fast !
I'm not sure we should rename it to `CheckRoundIcon`, even if it's a good suggestion.
Icon naming is decided by our design team.

Comment (tseho) [reply akeneo-design-system/src/icons/SuccessIcon.tsx]: Screenshot_2021-01-21_15-17-00

**Review by tseho — CHANGES_REQUESTED** [review]
we need to validate the icon naming before merging

**Review by tseho — APPROVED** [review]
New icon naming validated by Sophie, the notion page is being updated.
Thanks for the suggestion @ValentinMumble :coeuravecdoigts: 
Good job @tomglvng

File: akeneo-design-system/src/icons/SuccessIcon.tsx (hunk 1 above)
Comment (tomglvng) [reply akeneo-design-system/src/icons/SuccessIcon.tsx]: Design team thinks it is also a good suggestion and would like to keep CheckRoundIcon (https://www.notion.so/akeneo/Iconography-40ec9ee224094bd0906e05549b901ff1?p=fa1d77e9488546828fb1f1add5fd8c18).

**Pull request merged**